bertytech/content/features/index.ar.md

```diff
@@ -8,8 +8,7 @@ image: "berty_features.png"
 #footer_col1:
 #weight: -50
 aliases:
-  - /chat
-  - /messenger
+    - /chat
```

Comment:
I create a page for messenger, so I use /messenger for this page and I remove the alias to prevent redirection on /features

![image](https://github.com/berty/www.berty.tech/assets/33094556/58aadeef-0553-43c2-a389-ffa74c59344a)